src/.eslintrc.json

```diff
@@ -23,7 +27,10 @@
 				"sourceType": "script"
 			},
 			"rules": {
-				"no-unused-vars": "off"
+				"no-unused-vars": "off",
+				"indent": [ "error", "tab", {
+					"ignoredNodes": [ "Program > ExpressionStatement > CallExpression > FunctionExpression > *"	]
```

Comment:
Can you explain this line?

Reply:
As it was implemented according to the JSON grammar, I cannot add comments.

Unlike other codes, `wrapper.js` is implemented in UMD, and is implemented as follows.

```
// jQuery's UMD Code Style
// https://contribute.jquery.org/style-guide/js/#full-file-closures.
( function( factory ) {

    // Indentation: 1

} ( function() {

// Indentation: 0

} ) );
```

This style violates new eslint indentation rule.

```
// New eslint rule
( function( factory ) {

// Immediately Invoked Function Expression's indentation: 0

} ( function() {

    // Normal function's indentation: 1

} ) );
```

Therefore, `wrapper.js` required an specific exception.
I added "ignoredNodes" rule that indentation check is not performed for 1 depth of outer FunctionExpressions by analyzing the syntax tree of `wrapper.js` at https://esprima.org/demo/parse.html.

```
// Specific exception for UMD
( function( factory ) {

    var a;          // <-- ignore (1 depth)

    function b() {  // <-- ignore (1 depth)
        var c;      // <-- check! (2 depth ~)
    }

} ( function() {

var a;          // <-- ignore (1 depth)

function b() {  // <-- ignore (1 depth)
    var c;      // <-- check! (2 depth ~)
}

} ) );
```

I think it's for the best...

Reply:
You can add comments to ESLint config files! It’s not a true JSON, it’s “JSON with comments”. And we already use comments in some of them.

Reply:
I add comments.